Check translate

lomamech opened the pull request:
improvement(Translate Key)
- Fix and Improve translation
- Make all transactions make Sense

closes #2757

lomamech committed:
improvement(Translate Key)
- Fix and Improve translation

lomamech committed:
improvement(Translate key)
- Sanitaze and fix Error of translation

jniles commented on client/src/i18n/fr/cash_registry.json:
```diff
@@ -1,5 +1,5 @@
 {
     "CASH_REGISTRY" : {
-        "TITLE" : "Registre des payments"
+        "TITLE" : "Registre des paiments"
```
Ce n'est pas "paiements"?

jniles commented on client/src/i18n/fr/depot.json:
```diff
@@ -8,23 +8,23 @@
       "TITLE" : "Gestion des dépôts"
     },
-    "JOIN_LOCATION" : "Joindre locatlisation",
+    "JOIN_LOCATION" : "Joindre la localisation",
```
👍

jniles commented on client/src/i18n/fr/form.json:
```diff
@@ -680,26 +680,26 @@
+            "COUNTRY": "Sélectionner le pays",
             "CREDITOR": "Sélectionner un Créditeur",
-            "CREDITOR_GROUP": "Sélectionner Groupe de créditeurs",
+            "CREDITOR_GROUP": "Sélectionner un Groupe créditeur",
```
A mon avis, nous devons dire "Group Créditeur" tous en majescule, ou "group créditeur" tous en miniscule.

lomamech replied:
I preferred to use the first formulation "Groupe Créditeur"

jniles commented on client/src/i18n/fr/posting_journal.json:
```diff
@@ -1,9 +1,9 @@
-    "TRANSACTION_CURRENCY":"Monnaie Transaction",
-    "REMOVE_ROW" : "Supprimer Ligne(s)",
+    "TRANSACTION_CURRENCY":"Monnaie de la transaction",
+    "REMOVE_ROW" : "Supprimer la(les) ligne(s)",
```
👍

jniles reviewed (commented):
Looks great to me!

I made two comments to look at for clarity.  You are the expert here though.  Let me know what you think!

lomamech committed:
test(Complex Voucher)
- Update test E2E for complex voucher, update text from translate key
  modification

lomamech committed:
improvement(DebtorGroup Key)
- Use capital letter for the french translation for Debtor Group

lomamech committed:
Rebase with Branch Master
